application/forms/Config/GeneralConfigForm.php

```diff
@@ -335,6 +336,54 @@ public function createElements(array $formData)
                 )
             );
         }
+        if (isset($formData['grafana_accessmode']) && ( $formData['grafana_accessmode'] === 'iframe' )) {
+            $this->addElement(
+                'checkbox',
+                'grafana_jwtEnable',
+                array(
+                    'label' => $this->translate('Enable JWT'),
+                    'value' => false,
+                    'description' => $this->translate('Enable JWT. Grafana host will receive the JWT token to authorize the user.'),
+                    'class' => 'autosubmit',
+                )
+            );
+            if ($formData['grafana_jwtEnable']) {
```

Comment:
thorws an error in my setup, better:
`if (isset($formData['grafana_jwtEnable']) && $formData['grafana_jwtEnable']) {`
?
except this it works perfectly!